packages/website/docs/basic-options.md

```diff
@@ -3,18 +3,48 @@ id: basic-options
 title: Basic configuration options
 ---
 
-:::note Draft
+Everything you need to create fantastic autocomplete experiences.
 
-This page needs to cover:
+We've built Autocomplete to give you unlimited flexibility while freeing you from having to think about things like keyboard navigation, accessibility, or UI state. **The library offers a wide variety of APIs to let you fully customize the behavior and rendering of your autocomplete.**
 
-- There are only two required params to create an autocomplete:
-  - You need to provide a selector for the **container** you want autocomplete to appear in.
-  - You need to define where to get the items to display using **getSources** (or a **plug-in** which provides **getSources**). Check our **Sources** core concept for more information.
-- Beyond this, there are many parameters you can use to customize the experience and help you with development. Here are some commonly used ones:
-  - Use **placeholder** to define the text that appears in the input when a user hasn’t typed anything.
-  - Use **autoFocus** to focus on the search box when the page is loaded and **openOnFocus** to display items as soon as a user selects the autocomplete (without typing anything).
-  - Use the **onStateChange** hook to call a function whenever the state changes (see our **State** core concept for more info).
-  - Use **`debug: true`** to keep the autocomplete panel with items open, even when the blur event occurs. (This is only meant to be used during development. See our **Debugging guide** for more info.)
-- Check out the **API reference** for a full list of params.
+Yet, only two parameters are required to create an autocomplete:
+- The **container** you want your autocomplete to go in.
+- The **sources** from which to get the items to display (see more in [**Sources**](sources)).
 
-:::
+```js
+import { autocomplete } from '@algolia/autocomplete-js';
+
+autocomplete({
+  container: '#autocomplete',
+  getSources() {
+    return [
+      {
+        getItems() {
+          return [
+            { label: 'Twitter', url: 'https://twitter.com' },
+            { label: 'GitHub', url: 'https://github.com' },
+          ];
+        },
+        getItemUrl({ item }) {
+          return item.url;
+        },
```

Comment:
Even though it's technically not necessary for the autocomplete to work, it is for it to be keyboard navigable so I thought I'd keep it.